### DISCUSSION TITLE
Release v0.3.6

### DISCUSSION THREAD

**Discussion by alvin-reyes** [application-research/estuary#790] [Announcements]
## What's Changed
* feat: shuttle libp2p ping by @jcace in https://github.com/application-research/estuary/pull/718
* Staging zone refactor by @en0ma in https://github.com/application-research/estuary/pull/788
* Use ExpirableLRU to cache http requests with TTL by @neelvirdy in https://github.com/application-research/estuary/pull/789

**Full Changelog**: https://github.com/application-research/estuary/compare/v0.3.5...v0.3.6

<hr /><em>This discussion was created from the release <a href='https://github.com/application-research/estuary/releases/tag/v0.3.6'>Release v0.3.6</a>.</em>

**Comment by WinfredGe** [comment]
您发给我的信件已经收到。
